docs/build-ansible.rst

```diff
@@ -18,6 +18,8 @@ Setup for running from source
 
 antsibull uses the ``poetry`` tool to build and install::
 
+    # We have had reports of failed builds using macOS, you may want to consider using the ``python:3.9.1`` docker image if you are running macOS to build.
```

Comment:
I don't know if we want to call out `python:3.9.1` specifically (that's what I tested it on) or `python:3` or `python:3.9` or `python:latest`... any other thoughts / opinions?

Reply:
Is using a docker container practical ? I suppose we'd need to install not just the python deps in it but also git, twine, etc.

I think creating an issue with a description of the problem and then linking to that for context until we figure it out is a good start until we figure it out.

Reply:
If you weren't able to complete the build otherwise you might find it practical. :wink: It was just meant as a note to help people who might get stuck. Other than an initial step of something like `docker container run -it --rm -v $(pwd):/antsibull -w /antsibull python:3.9.1 /bin/bash` the rest of the doc reads the same. That particular image already has `git`, but you're right about `twine` if you were to take it that far, or if you shared the volume (like above) you could complete the other steps on your host OS.